### PR TITLE
Disable dynamic loading for help

### PR DESCRIPTION
**Pull request by cquiroz**
The help is broken since we switched to scala 3. Traced to
https://github.com/lampepfl/dotty/issues/15701

We'll do static loading

### CODE REVIEW

File: explore/src/main/scala/explore/ExploreLayout.scala [hunk 1]
```diff
@@ -27,12 +27,12 @@ import react.semanticui.modules.sidebar.SidebarPusher
 import react.semanticui.modules.sidebar.SidebarWidth
 
 import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportTopLevel
 
 final case class ExploreLayout(
-  c:        RouterCtl[Page],
```
Comment (toddburnside) [inline explore/src/main/scala/explore/ExploreLayout.scala:32]: Huh? We weren't using this anymore, eh? Good catch.

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/ExploreLayout.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/ExploreLayout.scala:32]: Yeah a bit surprising

**Comment by cquiroz** [comment]
Size before and after are comparable. the index file increased on 138k but that was on the separate dynamic bundle which was around 140k

**Pull request merged**